_source/_posts/2022-04-06-react-azure-functions.md

```diff
@@ -0,0 +1,416 @@
+---
+layout: blog_post
+title: "How to Build and Deploy a Serverless React App on Azure"
+author: nickolas-fisher
+by: contractor
+communities: [javascript]
+description: "A tutorial that shows you how to build and deploy a React app that securely calls serverless functions on Azure."
+tags: [react, azure, serverless, javascript]
+tweets:
+  - ""
+  - ""
+  - ""
+image:
+type: conversion
+github: https://github.com/oktadev/okta-react-azure-functions-example
+---
+
+Microsoft's Azure platform has as many high-tech products as anyone could ever want. Among these is the [Azure Static Web Apps](https://azure.microsoft.com/en-us/services/app-service/static/). As the name suggests, the platform hosts static web apps that don't require a back end. Azure supports React, Angular, Vue, Gatsby, and many more out of the box.
+
+However, you may run into a situation where you need a little back-end support. You can imagine where you might need the backend to run one or two API calls. For this task, Azure offers the [Functions](https://azure.microsoft.com/en-us/services/functions/) platform as well. Functions is a serverless computing platform that supports .NET, Node.js, Python, etc. It takes setting up a server, building logging and exception handling, and providing a high availability environment at a reasonable price.
```

Comment:
```suggestion
However, you may run into situations where you need a little back-end support, such as when you need the backend to run one or two API calls. For this task, Azure offers the [Functions](https://azure.microsoft.com/en-us/services/functions/) platform as well. Functions is a serverless computing platform that supports .NET, Node.js, Python, etc. It takes care of setting up a server, building logging and exception handling, and providing a high availability environment at a reasonable price.
```
conciseness. Also, should it say "takes care of"? - looks like the words are missing, but I may be wrong.